### PR TITLE
Version B - Usability Testing

### PR DESCRIPTION
**Pull request by nmakuch**

### CODE REVIEW

File: web/src/components/MobileCancel.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            d="M11.93 10.82l3.56-3.56a.79.79 0 0 0 0-1.11.8.8 0 0 0-1.12 0L10.82 9.7 7.26 6.15a.8.8 0 0 0-1.12 0 .79.79 0 0 0 0 1.11l3.55 3.55-3.55 3.55a.79.79 0 0 0 .55 1.34.8.8 0 0 0 .56-.23l3.55-3.54 3.55 3.55a.79.79 0 0 0 .56.23.77.77 0 0 0 .54-1.33z"
+          />
+        </g>
+      </g>
+    </svg>
+  )
+}
+
+MobileCancel.propTypes = {
+  circleColour: PropTypes.string, //left or right
```
Comment (pcraig3) [inline web/src/components/MobileCancel.js]: Don't need this comment here.

File: web/src/components/Calendar.js [hunk 2]
```diff
@@ -397,6 +398,10 @@ const removeDate = css`
   width: 1rem;
   height: 1rem;
 
+  img circle {
+    fill: red;
+  }
+
```
Comment (pcraig3) [inline web/src/components/Calendar.js]: Looks like this CSS isn't being used anymore. Is it?

File: web/src/pages/ConfirmationPage.js [hunk 3]
```diff
@@ -108,10 +96,9 @@ class ConfirmationPage extends React.Component {
             <Trans>What happens next?</Trans>
           </H2>
           <p>
-            <Trans>By</Trans> {respondBy},{' '}
             <Trans>
-              we’ll send you a new appointment. You will always be contacted at
-              least 3 weeks before your appointment.
+              IRCC will send you a new appointment. You will always be contacted
+              at least 3 weeks before your appointment.
```
Comment (pcraig3) [inline web/src/pages/ConfirmationPage.js]: If we're changing this sentence, should we change it in the email as well?

File: web/src/pages/ConfirmationPage.js [hunk 4]
```diff
@@ -108,10 +96,9 @@ class ConfirmationPage extends React.Component {
             <Trans>What happens next?</Trans>
           </H2>
           <p>
-            <Trans>By</Trans> {respondBy},{' '}
             <Trans>
-              we’ll send you a new appointment. You will always be contacted at
-              least 3 weeks before your appointment.
+              IRCC will send you a new appointment. You will always be contacted
```
Comment (pcraig3) [inline web/src/pages/ConfirmationPage.js]: Also, we can bring back our IRCCAbbr now that we're referring to IRCC again.

File: web/src/components/Calendar.js (hunk 2 above)
Comment (nmakuch) [reply web/src/components/Calendar.js]: Woops, I had removed that but then went back during some last minute changes before usability testing and it must have gotten in somehow :s nice catch

File: web/src/pages/ConfirmationPage.js (hunk 4 above)
Comment (nmakuch) [reply web/src/pages/ConfirmationPage.js]: true! good point

Comment (nmakuch) [reply web/src/pages/ConfirmationPage.js]: tried to implement it to the email as well but after reading up on the subject I don't think most email applications support pseudo-selectors.

**Pull request merged**